pkg/operator/ceph/object/objectstore.go

```diff
@@ -75,6 +84,10 @@ func deleteRealmAndPools(context *Context, preservePoolsOnDelete bool) error {
 	}
 	logger.Infof("Found stores %v when deleting store %s", stores, context.Name)
 
+	err = disableRGWDashboard(context)
+	if err != nil {
+		return errors.Wrapf(err, "failed to disable dashboard rgw")
```

Comment:
ditto

Reply:
More information here about which dashboard failed to be disabled?